deploy/template.yaml

```diff
@@ -0,0 +1,290 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  ipv-core-back
```

Comment:
We might want to expand the description if only to help new joiners etc

Reply:
Yep, We should probably expand what this repo is in the readme as well with explanations for each lambda